content/bn/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,136 @@
+[ফিচার গেট](/bn/docs/reference/command-line-tools-reference/feature-gates/) সক্ষম করুন এবং kubelet কনফিগারেশন ফাইলে `imageMaximumGCAge` ফিল্ডের জন্য একটি মান সেট করুন।
+
+মানটি কুবার্নেটিসের সময়কাল (duration) হিসেবে নির্ধারিত হয়। Kubelet কনফিগারেশন ফাইলে `imageMaximumGCAge` ফিল্ডের জন্য বৈধ সময় এককগুলি হলো:
+
+- "ns" ন্যানোসেকেন্ডের জন্য
+- "us" বা "µs" মাইক্রোসেকেন্ডের জন্য
+- "ms" মিলিসেকেন্ডের জন্য
+- "s" সেকেন্ডের জন্য
+- "m" মিনিটের জন্য
+- "h" ঘন্টার জন্য
+
+উদাহরণস্বরূপ, আপনি যদি `imageMaximumGCAge` ফিল্ডের মান `12h 45m` সেট করেন, তাহলে এর অর্থ হবে ১২ ঘন্টা এবং ৪৫ মিনিট।
+
+{{< note >}}
+এই ফিচারটি কিউবিলেট (kubelet) পুনরায় চালু হওয়ার (restart) পরেও ইমেজ ব্যবহারের তথ্য সংরক্ষণ করে না। যদি কিউবিলেট রিস্টার্ট হয়, তাহলে ট্র্যাক করা ইমেজের বয়স পুনরায় শূন্যে সেট হয়, যার ফলে কিউবিলেটকে সম্পূর্ণ `imageMaximumGCAge` সময় অপেক্ষা করতে হয়, এরপর ইমেজের বয়সের ভিত্তিতে গারবেজ কালেকশনের জন্য যোগ্যতা নির্ধারণ করা হয়।
+{{< /note>}}
+
+### কন্টেইনার গারবেজ কালেকশন {#container-image-garbage-collection}
+
+কিউবিলেট (kubelet) অব্যবহৃত কন্টেইনারগুলো গারবেজ কালেকশন করে নিম্নলিখিত ভ্যারিয়েবলগুলির ভিত্তিতে, যেগুলো আপনি সংজ্ঞায়িত করতে পারেন:
+
+* MinAge: এটি হল সেই ন্যূনতম বয়স, যার পরে কিউবিলেট একটি কন্টেইনার গারবেজ কালেক্ট করতে পারে। এটি নিষ্ক্রিয় করতে `0` এ সেট করতে হবে।
+* MaxPerPodContainer: প্রতিটি পডের জন্য মৃত কন্টেইনারের সর্বাধিক সংখ্যা। এটি নিষ্ক্রিয় করতে  ` 0` এ সেট করতে হবে।
+* MaxContainers: ক্লাস্টারের জন্য মৃত কন্টেইনারের সর্বাধিক সংখ্যা। এটি নিষ্ক্রিয় করতে কম ` 0` এ সেট করতে হবে।
+
+এই ভ্যারিয়েবলগুলির পাশাপাশি, কিউবিলেট অজানা এবং মুছে ফেলা কন্টেইনারগুলিকেও গারবেজ কালেক্ট করে, সাধারণত সবচেয়ে পুরনো কন্টেইনারগুলো প্রথমে মুছে ফেলে।
+
+`MaxPerPodContaine`r এবং `MaxContainers` একে অপরের সাথে সংঘর্ষে পড়তে পারে এমন পরিস্থিতিতে, যেখানে একটি পডে সর্বাধিক কন্টেইনার সংখ্যা (MaxPerPodContainer) ক্লাস্টারের অনুমোদিত মোট মৃত কন্টেইনারের সংখ্যা (MaxContainers) থেকে বেশি হতে পারে। এই পরিস্থিতিতে, কিউবিলেট সংঘর্ষ সমাধান করতে MaxPerPodContainer এর মান সমন্বয় করে। একটি খারাপ পরিস্থিতি হতে পারে MaxPerPodContainer কে `1` এ নামিয়ে আনা এবং সবচেয়ে পুরনো কন্টেইনারগুলো মুছে ফেলা। এছাড়াও, যে পডগুলি মুছে ফেলা হয়েছে, তাদের মালিকানাধীন কন্টেইনারগুলো MinAge এর চেয়ে পুরনো হলে মুছে ফেলা হয়।
+
+{{<note>}}
+কিউবিলেট (kubelet) শুধুমাত্র সেই কন্টেইনারগুলো গারবেজ কালেকশন করে, যেগুলোর এটি পরিচালনা করে।
+{{</note>}}
+
+## গারবেজ কালেকশন কনফিগার করা {#configuring-gc}
+
+আপনি যেসব কন্ট্রোলার রিসোর্স পরিচালনা করছে, তাদের জন্য নির্দিষ্ট অপশন কনফিগার করে রিসোর্সের গারবেজ কালেকশন টিউন করতে পারেন। নিম্নলিখিত পৃষ্ঠাগুলো আপনাকে গারবেজ কালেকশন কনফিগার করতে সাহায্য করবে:
+
+* [কুবার্নেটিস অবজেক্টের কাসকেডিং ডিলিশন কনফিগার করা](/bn/docs/tasks/administer-cluster/use-cascading-deletion/)
+* [শেষ হওয়া জবগুলোর ক্লিনআপ কনফিগার করা](/bn/docs/concepts/workloads/controllers/ttlafterfinished/)
+  
+## {{% heading "whatsnext" %}}
+
+* কুবার্নেটিস অবজেক্টের [মালিকানা সম্পর্কে আরও জানুন.](/bn/docs/concepts/overview/working-with-objects/owners-dependents/).
```

Comment:
Use the English version as reference `[মালিকানা সম্পর্কে আরও জানুন](/docs/concepts/overview/working-with-objects/owners-dependents/)।`